sympy/combinatorics/fp_groups.py

```diff
@@ -494,6 +496,120 @@ def elements(self):
+        if not self.is_polycyclic:
+            raise NotImplementedError("No methods currently implemented to"
+                                            "check if the given group is polycyclic")
+        # Compute the derived series.
+        derived_series = [self] + self.derived_series()
+        derived_series.insert(0, self)
+        len_derived_series = len(derived_series)
+
+        # Compute the polycyclic series by computing the
+        # subnormal series for every quotient G_i/G_i+1
+        # in the derived series and finally,
+        # lifting them to the original derived series.
+        pc_series = [derived_series[0]]
+        for i in range(0, len_derived_series-1):
+            # Compute the quotient group G_i/Gi+1.
+            q_grp = subgroup_quotient(self.free_group, self, derived_series[i], derived_series[i+1])
+            # Return a subnormal series.
+            _subnormal_series = _subnormal_series(self.free_group, q_grp)
+            # The images of Every subgroup in _subnormal_series
+            # can be lifted to the original group
+            gens = q_grp.generators
+            q_map = homomorphism(derived_series[i], q_grp, gens, gens)
+            for sub_group in _subnormal_series:
+                # Pre-image of subgroup in quotient map.
+                _sub_group = q_map.invert_subgroup(sub_group)
+                pc_series.append(_sub_group)
+            pc_series.append(derived_series[i+1])
+
+        return pc_series
+
+    def pc_group(self):
+        '''Returns a the corresponding Pc-Group'''
+        from sympy.combinatorics.pc_groups import PcGroup
+        polycyclic_series = self.compute_polycyclic_series()
+        # Consistent by default
+        return PcGroup(polycyclic_series)
+
+def subgroup_quotient(free_group, fp_grp, G, H):
+    '''
+    Compute the quotient group G/H
+    where, H is the subgroup of G.
+    '''
+    if not isinstance(fp_group, FpGroup):
+        raise ValueError("The parent group must be an instance"
+                                "of FpGroup")
+
+    if isinstance(G, list):
+        g_gens, g_rels, _gens = reidemeister_presentation(fp_group, G, rel=True)
+        _homomorphism = homomorphism(G, fp_group, g_gens, _gens, check=False)
+        g_gens = _gens
+        g_rels = _homomorphism(g_rels)
+    else:
+        g_gens = G.generators
+        g_rels = G.relators
+
+    if isinstance(H, list):
+        h_gens, h_rels, _gens = reidemeister_presentation(fp_group, H, rel=True)
+        _homomorphism = homomorphism(H, fp_group, h_gens, _gens, check=False)
+        h_gens = _gens
+        h_rels = _homomorphism(h_rels)
+    else:
+        h_gens = H.generators
+        h_rels = H.relators
+
+    q_relators = g_gens + h_rels
+    q_group = FpGroup(free_group, q_relators)
+
+    # Return the quotient group with presentation
+    # <G.generators|q_realtors>
+    return q_group
+
+def _subnormal_series(G):
+    '''
+    Computes the subnormal series for an abelian group.
```

Comment:
"Compute" and "a subnormal series". But then, this isn't just any subnormal series - it's a subnormal series with cyclic factor groups. Maybe the whole method should be called `_cyclic_subnormal_series`